doc/API.md

```diff
@@ -770,6 +771,24 @@ destination
 standard success or error response. See
 [#standard-responses](#standard-responses).
 
+#### /renter/downloadasync/___*siapath___ [GET]
+
+downloads a file to the local filesystem. The call will return immediately.
+
+###### Path Parameters [(with comments)](/doc/api/Renter.md#path-parameters-1)
+```
+*siapath
+```
+
+###### Query String Parameters [(with comments)](/doc/api/Renter.md#query-string-parameters-1)
```

Comment:
These links I think point to the /download/ example, need to swap the '1' for a '2'.

Also looks like the rename and upload stuff point to the same path parameters as well, those need to be updated and pointed to the correct place. (may as well since we're worried about it)